aspnet/mvc/overview/getting-started/introduction/accessing-your-models-data-from-a-controller.md

```diff
@@ -21,19 +21,19 @@ In this section, you'll create a new `MoviesController` class and write code tha
 
 In Solution Explorer, right-click the *Controllers* folder and then click **Add**, then **Controller**.
 
-![](accessing-your-models-data-from-a-controller/_static/image1.png)
+![Screenshot that shows the Solution Explorer window. The right click menu from the Controllers folder is open and Add is selected. Controller is selected in the sub menu.](accessing-your-models-data-from-a-controller/_static/image1.png)
 
 In the **Add Scaffold** dialog box, click **MVC 5 Controller with views, using Entity Framework**, and then click **Add**.
 
-![](accessing-your-models-data-from-a-controller/_static/image2.png)
+![Screenshot that shows the Add Scaffold dialog box. M V C 5 Controller with views, using Entity Framework is selected.](accessing-your-models-data-from-a-controller/_static/image2.png)
```

Comment:
Per MS writing style guide "dialog box" should simply be "dialog." I know the rest of this page uses "dialog box." Don't fix those instances, just fix your alt-texts.